caravel/assets/javascripts/SqlLab/components/QueryLog.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import * as Actions from '../actions';
+
+import QueryTable from './QueryTable';
+import { Alert } from 'react-bootstrap';
+
+class QueryLog extends React.Component {
+  render() {
+    const activeQeId = this.props.tabHistory[this.props.tabHistory.length - 1];
+    const queries = this.props.queries.filter((q) => (q.sqlEditorId === activeQeId));
+    if (queries.length > 0) {
+      return (
+        <QueryTable
+          columns={['state', 'started', 'duration', 'rows', 'sql', 'actions']}
+          queries={queries}
+        />
+      );
+    }
+    return (
+      <Alert bsStyle="info">
+        No query history yet...
```

Comment:
Let's just assume all strings exposed to users need to be wrapped somehow, otherwise there'll be todos everywhere...